polyfill/lib/duration.mjs

```diff
@@ -138,6 +138,9 @@ export class Duration {
   static from(...args) {
     return ES.CastDuration(...args);
   }
+  static fromNanoSeconds(arg) {
```

Comment:
Let's call this fromNanoseconds

Reply:
Duration.from<Nano|Micro|Milli>second() seems a total non-sequitur. I can agree that we could leave it out of the cast, but these functions seem very wrong.